### PR TITLE
BF(PY3): use except ... as   syntax instead of except ...,

### PR DESCRIPTION
**Pull request by yarikoptic**
mass BF was done via cmdline

```bash
git grep -l 'except.*, [a-z]\+:' | xargs sed -ie 's|\(except.*\), \([a-z]*:\)|\1 as \2|g'
```

I am not quite sure how those didn't pop out whenever testing was done on python3, but there you go ;-)

### CODE REVIEW

**Comment by ryansydnor** [comment]
+1 ship it!

**Comment by yarikoptic** [comment]
travis fails on 3.x pythons the same way as it started to fail (to install!) for me elsewhere:
```
  File "/home/travis/build/boto/boto/tests/unit/ec2/test_reservedinstance.py", line 2, in <module>

    from boto.ec2.connection import EC2Connection

  File "/home/travis/build/boto/boto/boto/__init__.py", line 1216, in <module>

    boto.plugin.load_plugins(config)

AttributeError: 'module' object has no attribute 'plugin'
```

actually it seems that all PRs from last 3 days had this error, but there were no related changes in the code... clues?

**Comment by ryansydnor** [comment]
Looks like this is being investigated more deeply here:

https://github.com/travis-ci/travis-ci/issues/5246

**Comment by yarikoptic** [comment]
Thanks for this reference... GCE.. Heh.  I could indeed try the PATH trick if someone wants such an obomination

**Comment by ryansydnor** [comment]
@yarikoptic - @jamesls fixed this issue here: https://github.com/boto/boto/pull/3439 

You should rebase and resubmit your PR.

**Comment by yarikoptic** [comment]
@ryansydnor yes sir, done sir!

File: boto/glacier/concurrent.py [hunk 1]
```diff
@@ -237,7 +237,7 @@ def _process_chunk(self, work):
             except self._retry_exceptions as e:
                 log.error("Exception caught uploading part number %s for "
                           "vault %s, attempt: (%s / %s), filename: %s, "
-                          "exception: %s, msg: %s",
+                          "exception: %s as msg: %s",
```
Comment (jamesls) [inline boto/glacier/concurrent.py]: I'm going to go ahead and undo this change, I don't think it was intentional.

**Comment by jamesls** [comment]
Merged, thanks!

File: boto/glacier/concurrent.py (hunk 1 above)
Comment (yarikoptic) [inline boto/glacier/concurrent.py]: d'oh -- sorry about this.  I did review the changes but managed to miss this one :-/
